frontend/src/scenes/persons/person-utils.ts

```diff
@@ -56,7 +56,7 @@ export function asDisplay(person: PersonPropType | null | undefined, maxLength?:
             : undefined)
     )?.trim()
 
-    return display ? midEllipsis(display, maxLength || 40) : 'Person without distinct_id'
+    return display ? midEllipsis(display, maxLength || 40) : 'Anonymous Person'
```

Comment:
Seems fine, I'd just say "Anonymous" though

```suggestion
    return display ? midEllipsis(display, maxLength || 40) : 'Anonymous'
```